dist/tags/framework/debugger.html

```diff
@@ -407,8 +452,33 @@
   var $root = document.getElementById('__tagsmith_debugger');
   var $open = $root.querySelector('.__tagsmith_debugger_open');
   var $container = $root.querySelector('.__tagsmith_debugger_container');
+  var $testSetup = $root.querySelector('#__tagsmith_debugger_test_setup');
   var $close = $root.querySelector('.__tagsmith_debugger_close');
 
+  var testSetupLogger;
+
+  /**
+   * Test framework setup.
+   */
+  var testSetup = function() {
```

Comment:
Is there any test code to assert these lines in this function? Because there is a possibility that someone will delete it by mistake in the future and it should detect it by unit test.

Reply:
Added test.